actionlist/__tests__/components/sideMenuSpec.js

```diff
@@ -13,19 +13,22 @@ jest.unmock('../../app/components/SideMenu');
 describe('SideMenu', () => {
   it('should render the inner tree component structure properly', () => {
     const middlewares = [];
-    const initialState = {};
+    const initialState = {
+      actionItemsReducer: {
```

Comment:
I meant there doesn't seem to be anywhere that's referencing `initialState.actionItemsReducer` and it's an empty object so there's nothing to test in it anyway.